gamesinfo/dragonage2.sh

```diff
@@ -0,0 +1,7 @@
+game_steam_subdirectory="Dragon Age II"
+game_nexusid="dragonage2"
+game_appid=1238040
+game_executable="__Installer/Origin/redist/internal/EAappInstaller.exe"
+game_protontricks=("xaudio2_7=native" "d3dcompiler_43" "d3dx9" "arial" "fontsmooth=rgb")
```

Comment:
Quick side note: I saw that "arial" and "fontsmooth=rgb" where added to all games, so I imagine they're tricks for MO2 itself. In the near future, you may want to consider adding them globally at [step/configure_steam_wineprefix.sh](https://github.com/rockerbacon/modorganizer2-linux-installer/blob/handover/step/configure_steam_wineprefix.sh)